Add section on challenges (non-responding reviewers) (#955)
ropensci/dev_guide: softwarereview_editor.pt.Rmd

  - Escreva um comentário agradecendo ao revisor;
  - Registre a revisão digitando um novo comentário `@ropensci-review-bot submit review <review-url> time <time in hours>`. Por exemplo, para a revisão [https://github.com/ropensci/software-review/issues/329#issuecomment-809783937](https://github.com/ropensci/software-review/issues/329#issuecomment-809783937) o comentário seria `@ropensci-review-bot submit review https://github.com/ropensci/software-review/issues/329#issuecomment-809783937 time 4`.
- Quando as alterações forem feitas, altere a _tag_ de status da revisão para `5/awaiting-reviewer-response` e solicite que os revisores indiquem a aprovação usando o [modelo de aprovação do revisor](#approval2template).
- Se as pessoas autoras tiverem a intenção de enviar um manuscrito de acompanhamento do tipo [*Applications*](https://besjournals.onlinelibrary.wiley.com/hub/journal/2041210X/features/applicationpapers) no periódico [Methods in Ecology and Evolution](https://besjournals.onlinelibrary.wiley.com/journal/2041210X), indique que o envio do manuscrito pode ser feito após a conclusão da revisão. 

#### Desafios durante a revisão {#challenges-during-review}

> - **Se o autor parar de responder** consulte [as políticas](#policies) e/ou envie um ping para os outros editores no canal do Slack para discussão. É importante ressaltar que, se um revisor foi atribuído a um problema fechado, entre em contato com ele ao fechar o problema para explicar a decisão e agradecê-lo mais uma vez pelo trabalho.  Informe os outros editores no canal Slack para que eles sejam considerados para um pacote no futuro com grandes chances de uma revisão de software tranquila (por exemplo, um autor de pacote que já tenha enviado pacotes para nós).

Reviewer: Hi! Just to clarify If I understood correcly:

"Let the other editors know in the Slack channel to consider them for a package in the future with high chances of smooth software review (e.g. a package author who has already submitted packages to us)."

I think I'm not sure if I understood the context of the whole sentence. In this case, "consider them" is related to the reviewer, right?

Author: Yes, that's right.

> - **Se um revisor estiver atrasado na revisão ou parar de responder** envie um lembrete após uma semana e novamente após duas semanas. O primeiro lembrete pode ser um `@tag` no GitHub. Depois disso, use e-mail ou outra forma de comunicação direta. Se depois de três semanas você ainda não tiver recebido resposta, determine a melhor maneira de seguir em frente sem eles:

Reviewer: ```suggestion
- **Se um revisor estiver atrasado na revisão ou parar de responder** envie um lembrete após uma semana e novamente após duas semanas. O primeiro lembrete pode ser um `@tag` no GitHub. Depois disso, use o e-mail ou outra forma de comunicação direta. Se depois de três semanas você ainda não tiver recebido resposta, determine a melhor maneira de seguir em frente sem o revisor:
```

>  If after 3 weeks there is still no response, determine how best to move ahead without them:

In this case, "without them" refers to the reviewer, right?

Author: Yes.  Perhaps I should change these in English, too, to be clearer.

  - Se o revisor já tiver enviado sua revisão primária e outro revisor estiver ativo e fornecendo feedback substancial, o editor poderá prosseguir com o processo de revisão e deverá assumir a função do revisor ausente para determinar se as alterações dos autores são suficientes.

>   - Se o revisor ausente não tiver enviado sua revisão, o editor deve tentar encontrar um novo revisor e prosseguir com o processo de revisão assim que duas revisões forem recebidas. Nesse ponto, o editor deve priorizar a busca de revisores experientes que possam se comprometer com uma resposta rápida. Não se esqueça de enviar mensagens para outros editores no Slack.

Reviewer: ```suggestion
  - Se o(a) revisor(a) ausente não tiver enviado sua revisão, o(a) editor(a) deve tentar encontrar uma nova pessoa revisora e prosseguir com o processo de revisão assim que duas revisões forem recebidas. Nesse ponto, o(a) editor(a) deve priorizar a busca de revisores(as) experientes que possam se comprometer com uma resposta rápida. Não se esqueça de enviar mensagens para outros(as) editores(as) no Slack.
```

>     - A seu critério, o editor *pode* optar por atuar como segundo revisor, mas só deve fazê-lo após várias tentativas fracassadas de encontrar um novo revisor e se o editor tiver experiência suficiente para isso. Não recomendamos que os editores façam isso com frequência, pois isso aumenta a carga de trabalho e reduz a diversidade de pontos de vista trazidos à comunidade pelos revisores.

Reviewer: ```suggestion
    - A seu critério, o(a) editor(a) *pode* optar por atuar como segundo(a) revisor(a), mas só deve fazê-lo após várias tentativas fracassadas de encontrar um(a) novo(a) revisor(a) e se o(a) editor(a) tiver experiência suficiente para isso. Não recomendamos que os(as) editores façam isso com frequência, pois isso aumenta a carga de trabalho e reduz a diversidade de pontos de vista trazidos à comunidade pelos(as) revisores(as).
```

>     - Em qualquer caso, faça um comentário agradecendo ao revisor original e remova-o com `@ropensci-review-bot remove @username from reviewers`.

Reviewer: ```suggestion
    - Em qualquer caso, faça um comentário agradecendo ao revisor(a) original e remova-o(a) com `@ropensci-review-bot remove @username from reviewers`.
```
